mhkit/river/io/usgs.py

```diff
@@ -120,7 +120,10 @@ def request_usgs_data(
 
     # Use handle_caching to manage cache
     cached_data, metadata, cache_filepath = handle_caching(
-        hash_params, cache_dir, write_json, clear_cache
+        hash_params,
+        cache_dir,
+        cache_content={"data": None, "metadata": None, "write_json": write_json},
+        clear_cache_file=clear_cache,
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args